examples/select_datasets.md

```diff
@@ -20,24 +20,25 @@
 
 `client.select_datasets(where='cluster', ...)`:
 ```python
->>> cluster_name = client.select_clusters().get_list()[30]['grouping_name']
+>>> dataset = client.select_datasets().get_list()[0]['uuid']
+>>> cluster_name = client.select_clusters(where='dataset', has=[dataset]).get_list()[0]['grouping_name']
 >>> cluster_datasets = client.select_datasets(where='cluster', has=[cluster_name])
 >>> assert len(cluster_datasets) > 0
 
 ```
 
 `client.select_datasets(where='gene', ...)`:
 ```python
->>> gene_symbol = client.select_genes().get_list()[0]['gene_symbol']
->>> gene_datasets = client.select_datasets(where='gene', has=[f'{gene_symbol} > 0'], genomic_modality='rna', min_cell_percentage=5.0)
+>>> gene_symbol = client.select_genes(where="modality",has=["rna"]).get_list()[0]['gene_symbol']
+>>> gene_datasets = client.select_datasets(where='gene', has=[f'{gene_symbol} > 1'], genomic_modality='rna', min_cell_percentage=5.0)
 >>> assert len(gene_datasets) > 0
 
 ```
 
 `client.select_datasets(where='protein', ...)`:
 ```python
 >>> protein_name = client.select_proteins().get_list()[0]['protein_id']
->>> protein_datasets = client.select_datasets(where='protein', has=[f'{protein_name} > 5000'], min_cell_percentage=5.0)
+>>> protein_datasets = client.select_datasets(where='protein', has=[f'{protein_name} > 1000'], min_cell_percentage=5.0)
```

Comment:
Can you say anything about the semantics of this value, whether is `5000` or `1000`? Or is it arbitrary, and the pipelines now return smaller numbers?